tests/validation/upgradecluster_test.go

```diff
@@ -0,0 +1,380 @@
+package validation
```

Comment:
Same thing here with the rest of the Go files. Looks like they need to be gofmt'd, goimports, etc.